src/agents/ipp_group.jl

```diff
@@ -740,7 +767,31 @@ function IPPGroup(input_filename::String, model_data::HEMData, id = DEFAULT_ID)
             model_data.index_d,
             model_data.index_t,
         ),
-        Dict()
+        Dict(),
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
+        [],
```

Comment:
Wow! It might be worth making the McCormick bounds their own struct and giving them a default constructor that initializes with empty vectors. In either case, there's probably a nice trick to make a vector of empty vectors of the right length and then splat them out ... @HaleyRoss, do you know how to do that offhand? In pseudo code I'm thinking something like `[ [] * n]...`.